app/uk/gov/hmrc/cataloguefrontend/service/CostEstimationService.scala

```diff
@@ -58,21 +61,65 @@ object CostEstimationService {
       Json.reads[DeploymentConfig]
   }
 
-  final case class CostEstimation(totalSlots: Int) {
+  final case class ServiceCostEstimate(forEnvironments: List[ServiceCostEstimate.ForEnvironment]) {
 
-    lazy val yearlyCostGbp: Double =
-      totalSlots * 650
+    lazy val summary: ServiceCostEstimate.Summary =
+      forEnvironments
+        .foldLeft(Summary.zero)((summary, forEnv) => summary + Summary(forEnv.slots, forEnv.yearlyCostGbp))
```

Comment:
Since you've already defined `+` for Summary: `forEnviroments.map(_.toSummary).fold(Summary.zero)(_ + _)`
But we could skip the `toSummary` and use `_.values` if `forEnvironments` was modelled as a map

Reply:
Will do, there is indeed a monoidal `foldMap` waiting to get out! 🤓